lib/logstash/filters/mutate.rb

```diff
@@ -288,6 +291,17 @@ def convert_float(value)
     return value.to_f
   end # def convert_float
 
+  def convert_boolean(value)
+    if value == "true"
+      return true
+    elsif value == "false"
+      return false
+    else
+      @logger.warn("Failed to convert #{value} into boolean. Acceptable values are \"true\" and \"false\".")
+      return value
```

Comment:
Yeah! I really don't see the else case as what I would expect if a conversion can not happen. I support the idea of using next. Otherwise this conversion can get ignored or wrongly ingested somewhere else I guess.